core/dns_test.go

```diff
@@ -62,43 +68,51 @@ func TestDNSLookupTXTNoServer(t *testing.T) {
 func TestDNSSEC(t *testing.T) {
```

Comment:
I thought the whole point of this PR is that we're not paying attention to DNSSEC status at the boulder level.  Delete this test.